openapi.yaml

```diff
@@ -36,15 +36,31 @@ servers:
   - url: "https://api.umd.io/v1"
 tags:
   - name: courses
-    description: This set of endpoints lets you get data about university courses and their sections. You can get one course or section at a time, several courses or sections at a time, or a list of all the courses. All the data is scraped from Testudo.
+    description: >-
+      This set of endpoints lets you get data about university courses and their
+      sections. You can get one course or section at a time, several courses or
+      sections at a time, or a list of all the courses. All the data is scraped
+      from Testudo.
+
```

Comment:
a general comment about the line splitting occurring in this file: is there a standard char limit per line for yaml files? If not, should we enforce one of our own? I'm definitely a fan of char limits (I follow python's 80 char limit pretty religiously, though most people aren't happy with it being that short) so I wouldn't be against enforcing our own, even if only colloquially between us (we all set a ruler in vscode at 100 chars or something).

Reply:
I generally stick to 80 characters. Also, the `>-` means the final string doesn't occur a newline (like they're not preserved) so
newlines here are just aesthetic.